### PR TITLE
C-style for loop statement is deprecated in Swift 3. 

### PR DESCRIPTION
**Pull request by gabrielgarza**
Modifying to a while loop to achieve the same goal.

### CODE REVIEW

**Comment by gabrielgarza** [comment]
Reference
https://github.com/apple/swift-evolution/blob/master/proposals/0007-remove-c-style-for-loops.md

**Comment by 1ec5** [comment]
A better way to accomplish this is with a for loop over a range.

File: MapboxStatic.swift [hunk 1]
```diff
@@ -234,11 +234,14 @@ public class MapboxStaticMap {
 
             var output = encodeCoordinate(coordinates[0].latitude) + encodeCoordinate(coordinates[0].longitude)
 
-            for var i = 1; i < coordinates.count; ++i {
+            var i = 1
+            while i < coordinates.count {
```
Comment (1ec5) [inline MapboxStatic.swift]: for i in 1..<coordinates.count

**Comment by gabrielgarza** [comment]
good point, closing this one and making new pull request

**Pull request closed without merging**